tests/sentry/integrations/gitlab/__init__.py

```diff
@@ -0,0 +1 @@
+from __future__ import absolute_import
```

Comment:
I think it'll need to look like this: https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/__init__.py